config/plugins.yml

```diff
@@ -3,6 +3,7 @@ loomio_webhooks:
 
 loomio_org_plugin:
   repo:         loomio/loomio_org_plugin
+  branch:       tests-for-sidebar-implementation
```

Comment:
FYI, my workflow when deploying plugin branches has been

- Deploy core, with this pointed to the plugin branch
- Merge the plugin branch into loomio_org_plugin/master
- Deploy core with this pointed to the plugin's master